### PR TITLE
fix: merge descendants of Tag for UI tests

### PR DESCRIPTION
**Pull request by ThibaultChariere**
## 📋 Changes description

Added a semantic on the Spark Tag to merge the descendant nodes for UI tests.

## 🤔 Context

When UI testing a `TagTinted` component, there was an issue trying to find the text inside of it. Indeed we need to merge the descendants of this composable to access the text inside.

## ✅ Checklist

- [ ] Link to GitHub issues it solves. 
- [x] I have reviewed the submitted code.
- [x] I have tested on a phone device/emulator.
- [x] If it includes design changes, please ask for a review `spark-design` GitHub team.

## 🗒️ Other info

Validated with @soulcramer 

[Contributing](https://github.com/adevinta/spark-android/blob/main/docs/contributing.md) has more information and tips for a great pull request.

### CODE REVIEW

**Review by MarinaRomanova — CHANGES_REQUESTED** [review]
Could you add var name please

**Comment by soulcramer** [comment]
What do you mean by var name?

**Comment by soulcramer** [comment]
LGTM but we'll need to add a unit test for this as well

File: spark/src/main/kotlin/com/adevinta/spark/components/tags/Tag.kt [hunk 1]
```diff
@@ -67,7 +68,9 @@ internal fun BaseSparkTag(
     content: @Composable RowScope.() -> Unit,
 ) {
     Surface(
-        modifier = modifier.sparkUsageOverlay(),
+        modifier = modifier
+            .semantics(true) {}
```
Comment (SimonMarquis) [inline spark/src/main/kotlin/com/adevinta/spark/components/tags/Tag.kt:72]:
```suggestion
            .semantics(mergeDescendants = true) {}
```

**Review by MarinaRomanova — APPROVED** [review]

**Pull request merged**